tests/console/velociraptor_client_event_collect.pm

```diff
@@ -104,15 +100,26 @@ sub run {
         assert_script_run "chmod +x /tmp/genfile$i.sh";
```

Comment:
We are only triggering `SUSE.Linux.Events.ExecutableFiles` here. We will need to test the other artifacts too.